install.sh

```diff
@@ -13,6 +13,7 @@ SCRIPTS="$MHN_HOME/scripts/"
 cd $SCRIPTS
 
 if [ -f /etc/redhat-release ]; then
+    export PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
```

Comment:
I debated this. PATH inside the script shouldn't affect anything outside the script. Setting it to the clean default setting seemed best. In most environments appending the existing $PATH will just duplicate the path making it $PATH=$PATH:$PATH. But both work. I'll make the change.